### PR TITLE
fixed issues and added buttons

### PR DESCRIPTION
**Pull request by d3rpp**

### CODE REVIEW

File: src/lib/components/headers/landing-header.svelte
Comment (d3rpp) [inline src/lib/components/headers/landing-header.svelte:1]: The reason behind this shift is that the dropdown menu doesn't work when its trigger is in absolute positioning.

So I moved things around to make the position normal, and now the dropdown works. It should look the same.

Comment (JSpring0303) [reply src/lib/components/headers/landing-header.svelte:1]: excellent new sire, i shall sound the horns of triumph

Comment (d3rpp) [reply src/lib/components/headers/landing-header.svelte:1]: UPDATE: IT DOES NOT GIVE ME ONE MINUTE

Comment (JSpring0303) [reply src/lib/components/headers/landing-header.svelte:1]: no horns >:(

Comment (d3rpp) [reply src/lib/components/headers/landing-header.svelte:1]: resolved in new commit

Comment (d3rpp) [reply src/lib/components/headers/landing-header.svelte:1]: > no horns >:(

You may now sound the horns of triumph

**Review by JSpring0303 — DISMISSED** [review]
Who am I to judge the coding of the father

**Pull request merged**